### ISSUE TITLE
Quick actions - resched but no done

### ISSUE THREAD

**Issue by licaon-kter** [orgzly/orgzly-android#302]
Have a task:
* sched at 10 am
* Resched at .+5h

The notification that popups up at 10am has only *RESCHED* and *SNOOZE* options, so I can't mark it as DONE

**Comment by nevenz** [comment]
"Reschedule" from notification does what marking note as DONE from app would (it shifts the time).

Do you need to actually change the state of note to DONE? Even from the app that's not possible currently without first removing the repeater.

**Comment by licaon-kter** [comment]
I'm not sure why I would not have the option, yes I have a resched, but if I've *done* the thing...I'd like to mark it as such.

Now, I do get the point, if I marked it DONE does the app cancel the re-sched or not?

**Comment by nevenz** [comment]
> I'm not sure why I would not have the option, yes I have a resched, but if I've done the thing...I'd like to mark it as such.

I agree it should be possible. I wonder if any Org mode user is doing this, and how.

> Now, I do get the point, if I marked it DONE does the app cancel the re-sched or not?

What do you mean by "cancel the re-sched"?

**Comment by licaon-kter** [comment]
I mean, while I would like the re-sched to be cancelled (eg. not show a new notification at .+5h) if I mark the task as done, I do understand that this might not be the wish of other users.

**Comment by iron-s** [comment]
There might be other done states, like FINISHED or CANCELED that probably shouldn't trigger rescheduling
